test/test_common.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import os
+
+
+def registered_packages():
+    base_dir = os.path.dirname(__file__)
+    pkg_dir = os.path.join(base_dir, '..', 'packages')
+    packages = [name for name in os.listdir(pkg_dir)
+                if os.path.isdir(os.path.join(pkg_dir, name))]
+    return packages
+
+
+@pytest.mark.parametrize('name', registered_packages())
+def test_package_imports(selenium, name):
+    selenium.load_package(name)
+    selenium.run('import %s' % name)
```

Comment:
This is a nice test to have, but it won't work in all cases.  The package name and the Python module path aren't always the same, see `python-dateutil`/`dateutil`.  What conda does is it allows the `meta.yaml` to specify a smoke test, e.g.:

```
test:
  imports:
    - dateutil
```

Reply:
You are right, will remove it form this PR and added a separate PR to parse the `test` section of `meta.yaml` in https://github.com/iodide-project/pyodide/pull/98